### PR TITLE
Softmax_cross_entropy op add axis

### PR DESCRIPTION
**Pull request by heavengate**
Add Attr(axis) for softmax_with_cross_entropy op

红框中为修改文档
softmax_cross_axis1
softmax_cross_axis2

### CODE REVIEW

**Comment by heavengate** [comment]
plz check doc @shanyi15

**Comment by shanyi15** [comment]
image
image

**Comment by heavengate** [comment]
sce

Done. @shanyi15

**Review by qingqing01 — DISMISSED** [review]

**Review by shanyi15 — DISMISSED** [review]
LG comments

File: python/paddle/fluid/tests/unittests/test_layers.py [hunk 1]
```diff
@@ -1225,6 +1225,8 @@ def make_softmax_with_cross_entropy(self):
             return (softmax)
             loss = layers.softmax_with_cross_entropy(x, y)
             return (loss)
+            loss = layers.softmax_with_cross_entropy(x, y, axis=0)
```
Comment (XiaoguangHu01) [inline python/paddle/fluid/tests/unittests/test_layers.py]: return之后的测试用例是否会被执行？建议再增加一些tensor rank>2的单测用例

Comment (heavengate) [reply python/paddle/fluid/tests/unittests/test_layers.py]: Done, Thanks.

**Review by shanyi15 — DISMISSED** [review]

**Review by XiaoguangHu01 — DISMISSED** [review]

**Review by shanyi15 — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
